### PR TITLE
Refactor EXP get Preferences

### PR DESCRIPTION
**Pull request by CarLosVegga**
Hi everyone!

This is another PR.

In general the changes I made include:

Created the query for getting the feedback areas where Experience Makers want to be rated in **queries/experience.js**
- (distance is excluded, that is out of their control)

Deleted 

I call the query for getting those feedback areas in **routes/experience.js**

In **ExperienceInfo.jsx** I decided to include the solution for that problem where after rating an experience it was not automatically displaying the rated label, so I could have in this branch the most functional version and I didn't have to wait until all PR were approved.

**Experience.jsx** was about calling my endpoint for getting feedback areas and use the useContext to pass it down.

On **Feedback.jsx** I consume the the context, I pass down the active and inactiveIDs array, and I am pretending to retrieve the information of those IDs with the feedback context (that will be in the following PR that looks into that).

On the **useSettings.jsx** I just modify the URL that will be in charge of making the requests.

### CODE REVIEW

File: back/routes/experience.js [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+router.get('/preferences/all', async (req, res) => {
+  const allFeedbackInfo = await AllFeedbackInfoQuery();
+  res.status(200).send(allFeedbackInfo);
+});
+
+/* 
+---------------------------
+---------------------------
+---------------------------
+OLD VERSION
```
Comment (vqdo) [inline back/routes/experience.js:19]: will this be removed eventually?

File: capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx [hunk 2]
```diff
@@ -67,7 +64,7 @@ export default function ExperienceInfo({ onUpdateRestaurants }) {
     if (!isRated) {
       setNewReview(resetRatingForm);
     }
-  }, [currentRestaurant, setNewReview]);
+  }, [currentRestaurant, setNewReview, isRated]);
```
Comment (vqdo) [inline capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx:67]: thanks for fixing this!

**Review by vqdo — APPROVED** [review]

File: back/routes/experience.js (hunk 1 above)
Comment (CarLosVegga) [reply back/routes/experience.js:19]: Yes! That was just for reference on which endpoints were not still updated

File: capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx (hunk 2 above)
Comment (CarLosVegga) [reply capstone/src/components/User/Adventurer/ExperienceInfo/ExperienceInfo.jsx:67]: Thanks for bringing that up to my attention!

**Pull request merged**